### DISCUSSION TITLE
Welcome to arc_platform Discussions!

### DISCUSSION THREAD

**Discussion by banta** [African-Ruby-Community/arc_platform#38] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by banta** [comment]
@kamalogudah, see the test failing below from PR #35 

![image](https://user-images.githubusercontent.com/668742/218402517-6b19fe10-5218-4528-9ea9-7153b5d15974.png)

**Reply by banta** [reply]
@kamalogudah, it's not failing because of CSS and JS.

**Reply by kamalogudah** [reply]
initially it was because of the css, this was the other error.
